src/webinars/2024/node-red-dashboard-multi-user.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Multi User Dashboards with Node-RED Dashboard 2.0
+subtitle: Join Joe Pavitt, lead developer on Node-RED Dashboard 2.0, as he discusses the new Multi User Dashboards feature, and how to get started with it.
```

Comment:
```suggestion
subtitle: Join Joe Pavitt, lead developer on Node-RED Dashboard 2.0, as he discusses the new Personalized Multi User Dashboards feature, and how to get started with it.
```

Reply:
@joepavitt can you perform a `git grep` on `Multi User` and prefix them all with Personalized?